src/components/textarea/example-textarea-with-word-limit.njk

```diff
@@ -0,0 +1,20 @@
+{% from "components/textarea/_macro.njk" import onsTextarea %}
+
+{{
+    onsTextarea({
+        "id": "textarea-word-limit",
+        "name": "feedback-limited",
+        "width": "30",
+        "label": {
+            "text": "Please provide some feedback",
+            "description": "For example, describe any difficulties you experienced in the use of this service"
+        },
+        "wordLimit": {
+            "limit": 5,
+            "wordCountSingular": "You have {x} word remaining",
+            "wordCountPlural": "You have {x} words remaining",
+            "wordCountOverLimitSingular": "You are {x} word over the limit",
```

Comment:
Suggestion: I think we should include the limit number in these 'over limit' warnings i.e. 'You are {x} words over the {limit} word limit' so users have a quick reference of max words allowed - this could be especially useful for people copying & pasting text in.

Happy for this to be considered as a separate enhancement though so we can progress this PR.